darwin_py.egg-info/PKG-INFO

```diff
@@ -0,0 +1,261 @@
+Metadata-Version: 2.1
```

Comment:
I think we should not version this file. If you agree, please remove it from the staged files with:
`git rm --cached darwin_py.egg-info/PKG-INFO`